celer/homotopy.py

```diff
@@ -245,7 +256,11 @@ def celer_path(X, y, pb, eps=1e-3, n_alphas=100, alphas=None,
 
     # do not skip alphas[0], it is not always alpha_max
     for t in range(n_alphas):
-        alpha = alphas[t]
+        # for enet use convention: lambda = alpha * l1_ratio
+        # and mu = 2 * alpha * (1 - l1_ratio)
+        alpha = alphas[t] * l1_ratio
```

Comment:
this is a dangerous design : it modifies the alpha that is passed to other solvers and can introduce issues if the check that l1_ratio is 1 in that case is done done properly.

Instead, just pass l1_ratio and alphas[t] to `celer `

Reply:
You are right! I didn't consider that.